src/lib/logger.ts

```diff
@@ -0,0 +1,19 @@
+import winston, { format } from 'winston'
+
+const { combine, timestamp, prettyPrint } = format
+
+export const createBaseLogger = (path: string) => {
+  const formats = [
+    winston.format.json(),
+    timestamp(),
+    process.env.NODE_ENV === 'development' ? prettyPrint() : undefined,
+  ].filter((v) => !!v) as winston.Logform.Format[]
+
+  return winston.createLogger({
+    format: combine(...formats),
+    transports: [new winston.transports.Console()],
```

Comment:
nit: suggest silencing logs in tests:

```suggestion
    transports: [new winston.transports.Console({
      silent: env.NODE_ENV === 'test'
    )],
```